File: app/src/main/java/com/karthik/splash/Models/PhotoDetail/Links.java
package com.karthik.splash.Models.PhotoDetail;


import com.google.gson.annotations.SerializedName;


public class Links{

	@SerializedName("portfolio")
	private String portfolio;

	@SerializedName("self")
	private String self;

	@SerializedName("html")
	private String html;

	@SerializedName("photos")
	private String photos;

	@SerializedName("likes")
	private String likes;
}

File: app/src/main/java/com/karthik/splash/models/photodetail/Links.kt
package com.karthik.splash.models.photodetail


import com.google.gson.annotations.SerializedName


class Links(@SerializedName("portfolio")
            private val portfolio: String? = null,
            @SerializedName("self")
            private val self: String? = null,
            @SerializedName("html")
            private val html: String? = null,
            @SerializedName("photos")
            private val photos: String? = null,
            @SerializedName("likes")
            private val likes: String? = null)